### PR TITLE
Added a reference link for the Adopter file to the WasmEdge's README file.

### PR DESCRIPTION
**Pull request by MeenuyD**
Added a reference link for the Adopter file to the WasmEdge's README file.The URL of the Adopter file is (https://wasmedge.org/docs/contribute/users).
#2970

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: README.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ If you have any questions, feel free to open a GitHub issue on a related project
 * Slack: Join the #WasmEdge channel on the [CNCF Slack](https://slack.cncf.io/)
 * Twitter: Follow @realwasmedge on [Twitter](https://twitter.com/realwasmedge)
 
+## Adopters
+
+Check out our [list of Adopters](https://wasmedge.org/docs/contribute/users/) who are using WasmEdge in their projects.
```
Comment (alabulei1) [inline README.md:74]: Can we add one sentence to call for other people to add their name if they're using WasmEdge?

Comment (MeenuyD) [reply README.md:74]: Can you please review now

Comment (MeenuyD) [reply README.md:74]: Ok

Comment (alabulei1) [reply README.md:74]: Sorry I may make you confused. The sentence you added is good. But I don't think we need two headlines for Adopter. Can you add the sentence behind `Check out our [list of Adopters](https://wasmedge.org/docs/contribute/users/) who are using WasmEdge in their projects.`?

**Review by alabulei1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MeenuyD) [reply README.md:74]: It is oky

**Comment by alabulei1** [comment]
@hydai Please merge this PR when you're free.

**Pull request merged**